pkg/controller/rest/didexchange/models.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright SecureKey Technologies Inc. All Rights Reserved.
+Copyright Avast Software. All Rights Reserved.
```

Comment:
While updating existing code : I would suggest adding `Copyright Avast Software. All Rights Reserved.` as newline instead of replacing existing license .

Reply:
Ok

Reply:
@sudeshrshetty I guess that after making your suggestion `repolint` job fails now or I have missed something else. Can you pls check it?